CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
This article is titled "Contributor Covenant Code of Conduct", and here we are talking about  "contributors" and also "maintainers".  Maybe these roles are not well-known by anyone reading this, specially those not used to contribute in big projects with many developers. 

I think that describing these roles may help on this. Or maybe it is enough to just add a link to [Anatomy of an open source project](https://opensource.guide/how-to-contribute/#anatomy-of-an-open-source-project), where these concepts are defined.

Reply:
Added the suggested link where these roles are described. Nice suggestion